src/schemas/subject.rs

```diff
@@ -0,0 +1,42 @@
+use std::collections::HashMap;
+
+use serde::{Deserialize, Serialize};
+use serde_json::Value;
+
+#[derive(Serialize, Deserialize, Debug, Clone, Eq, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct Subject {
+    /// "The subject's name that is displayed when processing this certificate"
+    pub display_name: String,
+    /// "Contact information of the subject"
+    pub contact: Contact,
+    /// "The legal entity of the subject"
+    pub legal_entity: Option<LegalEntity>,
```

Comment:
1. Schema also allows additional properties in the subject itself. 
2. However, I don't quite get idea to put `other` props in every possible place. Is it made because we want to pass this information regardless during the verification?

Reply:
I was told to add placeholder for additional fields in.
The reason for it was the fact it will be useful when displaying details of a specific Golem certificate.

Reply:
Yeah I said to add a map that collects all additional fields and not have legal entity separately because it is an example of a possibly useful additional field.

@MrDarthShoe we do not need to add it to everywhere but when we are printing the details of the certificate, we should print the extra information also. In the list of certificates, only `display_name` would show up.